### PR TITLE
[risk=no] Make paginator work

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**

### CODE REVIEW

File: public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx [hunk 1]
```diff
@@ -80,13 +90,28 @@ export class VariantTableComponent extends React.Component<Props, State> {
...
+
+    handlePageClick = (data) => {
+        const {searchTerm} = this.props;
+        this.setState({loading: true, page: data.selected + 1, currentPage: data.selected + 1});
+        genomicsApi().searchVariants(searchTerm, data.selected + 1).then(
+                results => {
+                    this.setState({
+                        searchResults: results.items,
+                        loading: false
+                    }, () => { window.scrollTo(0, 0); });
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx]: I think we should scroll to the top of the table and not the top of the page. It is disorienting to the user.
Also maybe add a spinner when the next 50 load.

Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx]: Changing the scroll to table top. Added the spinner.

File: public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx [hunk 2]
```diff
@@ -117,10 +145,9 @@ export class VariantTableComponent extends React.Component<Props, State> {
                         containerClassName={'pagination'}
                     />}
 
-            </div> : <div style={styles.tableFrame}> {loading && <div style={styles.center}><Spinner /> </div>}</div>
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx:126]: I think the spinner should be inside the tableFrame. 
I think the scroll to the top of the table works fine to communicate the user that the page has changed

**Review by mhigbyflowers — COMMENTED** [review]
I've noticed when a variant is expanded, the expansion doesn't close and expands a variant that is in the same location that the previous one was.

File: public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx (hunk 2 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx:126]: @mhigbyflowers Changing the spinner to its original place. Let me know if you think we have to remove spinner on the top while the page results load.

**Comment by SrushtiGangireddy** [comment]
> I've noticed when a variant is expanded, the expansion doesn't close and expands a variant that is in the same location that the previous one was.

Addressing this in another pr.

File: public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx (hunk 2 above)
Comment (mhigbyflowers) [reply public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx:126]: @SrushtiGangireddy  yeah, lets remove that top spinner all together

Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx:126]: Okay, removed it.

**Review by mhigbyflowers — APPROVED** [review]
👍

**Pull request merged**